docs/azure.md

```diff
@@ -68,6 +68,27 @@ For more advanced scenarios, please check the [terraform-dcos documentation for
 
 ### Kubernetes configuration
 
+#### RBAC
+
+**NOTE:** By default, it will provision a Kubernetes cluster without [RBAC](https://docs.mesosphere.com/services/kubernetes/1.1.1-1.10.4/authn-and-authz/#rbac) support.
```

Comment:
@rimusz perhaps `s/it/dcos-kubernetes-quickstart/`.

Reply:
Why are we repeating text in all docs?